Lib/test/test_capi/test_opt.py

```diff
@@ -1923,6 +1923,19 @@ def testfunc(n):
         self.assertNotIn("_GUARD_TOS_INT", uops)
         self.assertIn("_CALL_LEN", uops)
 
+    def test_get_len(self):
+        def testfunc(n):
+            a = [1, 2, 3, 4]
+            for _ in range(n):
+                match a:
+                    case [1,2]:
+                        _ = len(a) - 1
```

Comment:
Could this be something like `pass` instead? The `GET_LEN` instruction comes from the pattern so perhaps the body of the case statement can be simplified?
```suggestion
                        pass
```